dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSPChannelTest.java

```diff
@@ -203,6 +209,122 @@ public void testIgnoreWriteAfterCommitFalse() throws Exception {
 
+    /**
+     * Tests that ignoreWriteAfterCommit=true does NOT prevent connection closure for plain IOExceptions.
+     * 
+     * This test verifies that ignoreWriteAfterCommit is specific to MessageSentException
+     * and doesn't apply to general IOExceptions.
+     */
+
+    @Test
+    @AllowedFFDC("java.lang.IllegalStateException")
+    @AllowedFFDC("java.io.IOException")
+    public void testIgnoreWriteAfterCommitTrueWithIOException() throws Exception {
+        Socket socket = null;
+        boolean socketExceptionOccurred = false;
+        
+        try {
+            updateHTTPOptions(true); // Set ignoreWriteAfterCommit=true
+
+            String address = server.getHostname() + ":" + server.getHttpDefaultPort();
+            
+            // Create a persistent connection
+            socket = new Socket(server.getHostname(), server.getHttpDefaultPort());
+            socket.setSoTimeout(10000); // 10 seconds timeout
+            socket.setKeepAlive(true);
+            
+            // Send a request for a slow responding JSP
+            String abruptRequest = "GET /" + APP_NAME + "/slow-response HTTP/1.1\r\n" +
+                                  "Host: " + address + "\r\n" +
+                                  "Connection: keep-alive\r\n" +
+                                  "\r\n";
+            OutputStream os = socket.getOutputStream();
+            os.write(abruptRequest.getBytes());
+            os.flush();
+
+            BufferedReader reader = new BufferedReader(new InputStreamReader(socket.getInputStream()));
+            
+            
+            String line;
+            boolean foundStartMarker = false;
+            //boolean headersRead = false;
+            while ((line = reader.readLine()) != null) {
+                LOG.info("< "+line);
+                if (line.contains("Beginning slow response...")) {
+                    foundStartMarker = true;
+                    break;
+                }
+            }
+
+            //Check to see if the response has been started
+            Assert.assertTrue("Should find start marker in response", foundStartMarker);
+
+             // Now read a small portion of the body to ensure the response is in progress
+            char[] buffer = new char[1024];
+            int charsRead = reader.read(buffer, 0, buffer.length);
+            LOG.info("Read " + charsRead + " chars of response body: " + 
+                              new String(buffer, 0, charsRead));
+            
+            // Wait a bit to ensure the server is in the middle of sending data
+            LOG.info("Waiting 2 seconds before triggering broken pipe...");
+            Thread.sleep(2000);
+           
+
+            // Now shutdown the stream, but keep the socket open to cause the server's write operations to fail
+            LOG.info("Shutting down stream to trigger broken pipe");
+            socket.shutdownInput();
+            
+            socket.shutdownOutput();
+            
+            
```

Comment:
There seems to be a lot of extra white space in this test class if you could clean that up it would be great. Thanks!